presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -46,33 +43,6 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
-    @Nullable
-    public String getConnectionUser()
-    {
-        return connectionUser;
-    }
-
-    @Config("connection-user")
-    public BaseJdbcConfig setConnectionUser(String connectionUser)
-    {
-        this.connectionUser = connectionUser;
-        return this;
-    }
-
-    @Nullable
-    public String getConnectionPassword()
-    {
-        return connectionPassword;
-    }
-
-    @Config("connection-password")
-    @ConfigSecuritySensitive
-    public BaseJdbcConfig setConnectionPassword(String connectionPassword)
-    {
-        this.connectionPassword = connectionPassword;
-        return this;
-    }
-
     @Nullable
     public String getUserCredentialName()
```

Comment:
this could be also extracted as wrapping credential provider (`ExtraCredentialsCredentialProvider`). Please do it as separate commit on top of this one.